DESCRIPTION

```diff
@@ -58,9 +58,11 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Collate:
-    'T_and_F_symbol_linter.R'
+    'make_linter_from_xpath.R'
+    'xp_utils.R'
     'utils.R'
-    'aaa.R'
+    'AAA.R'
```

Comment:
somewhat surprisingly roxygen2 uses C sorting -> `aaa.R` was coming _after_ `T_and_F_symbol_linter.R`, which seems unintentional. Hence the name change.